spacecmd/spacecmd.changes

```diff
@@ -1,3 +1,7 @@
+Tue July 19 09:20 AM CEST 2022 - ssnow@suse.com
```

Comment:
Line 1 and 2 should be removed (they're added when the package is released). Line 3 and 4 are enough.

Reply:
ok, I was able to make the change by myself :). Before merging, please wait python team or @meaksh review